Sources/ListLabel25.pas

```diff
@@ -1954,6 +2000,32 @@ procedure TListLabel25.DoExport(Wnd: HWND; ProjectFile, OriginalProjectFile: Str
           ok := CheckError(LLPrintOptionsDialog(CurrentJobHandle, Wnd, PChar(FAutoDialogTitle))) = CE_OK;
         end;
 
+        if ExportFile.Length > 0 then
+        begin
+          char1 := '1';
+          optionStrPath := 'Export.Path';
+          optionStrFile := 'Export.File';
+          optionStrQuiet := 'Export.Quiet';
+          optionStrShow := 'Export.ShowResult';
+          if SameText(ExportFormat, 'PDF') then
+          begin
+            LlPrintSetOptionString(JobHandle, LL_OPTION_ENHANCED_SKIPRETURNATENDOFRTF, PChar(char1));
```

Comment:
I'd avoid to implicitly change options based on the export format. This might be unexpected for other users. Also, `LL_OPTION_... ` is not a print option, so it would need to be set using `LlSetOption` (and can safely be set before printing/exporting in user code then).

Reply:
Your right, I remove it. It was copied code from am project.